pkg/core/aliaslist_test.go

```diff
@@ -0,0 +1,304 @@
+/* Copyright 2022 Zinc Labs Inc. and Contributors
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+package core
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
```

Comment:
We use `github.com/stretchr/testify/assert` in test, not use require. we should keep it.

Reply:
`assert` allows execution to continue after a failed assertion, `require` avoids that, i think it is better